src/App.js

```diff
@@ -1,67 +1,68 @@
 import React, { Component } from 'react';
-import firebase, { auth } from './FirebaseConfig.js';
+// import firebase, { auth } from './FirebaseConfig.js';
 import { PageContent } from './Enums.js';
 import PageContainer from './PageContainer.js';
 import 'typeface-roboto';
-import SignUpInController from './SignUpIn/SignUpInController.js';
+// import SignUpInController from './SignUpIn/SignUpInController.js'
 
 // The main entry page to load when user is not signed in.
 // Currently (win18), it is just the first page of sign in/up (select account type).
 // Potentially, it would be “About” page in the future.
 // For now, rendering sign up/in components
 
 class App extends Component {
-
     constructor(props) {
         super(props);
 
         this.state = {
             // TODO: a hack to prevent showing logged out page first.. better way?
-            authenticated: false,
-            account: null
+            authenticated: true,
+            account: 'Sarah'
         };
     }
 
-    componentDidMount() {
-        // check whether user is logged in
-        auth.onAuthStateChanged(function(user) {
-            if (user) {
-                // grab user's account object
-                var accountRef = firebase.database().ref('accounts').child(user.uid);
-                accountRef.once('value').then(function(snapshot) {
-                    this.setState({
-                        authenticated: true,
-                        account: snapshot.val()
-                    });
-                }.bind(this));
-            } else {
-                this.setState({
-                    authenticated: true,
-                    account: null
-                });
-            }   
-        }.bind(this));
-    }
```

Comment:
I'm assuming you had to comment all this out to bypass log in? SignUpIn is finally merged into master so please log in with that.